.devops/templates/tools.yml

```diff
@@ -2,8 +2,8 @@
 steps:
   - task: NodeTool@0
     inputs:
-      versionSpec: '14.x'
-      checkLatest: true
+      versionSpec: '14.18.1'
```

Comment:
using volatile ranges on CI is not a very good idea in terms of getting repro of particular issue. this uses lowest supported version of node that reflects our engines definition

Reply:
Why not use a fixed version everywhere to avoid possible different behavior between local and CI?

Reply:
aim of this was to break as less as possible users local environments. If I set this strictly to engines, I guess lot of people would need to re-install node to that particular version.

This is good enough approach IMO. have set baseline. 

If you have more reasoning or suggestion please share that.

Reply:
I assumed so, just double checking 👍🏻